src/routing/index.html

```diff
@@ -1,8 +1,8 @@
 ---
 title: Routing
 description: |
-  Content routing is the way to determine where to find a given CID on the network;
-  specifically, which network peers provide specific CIDs. 
+  Routing is the way to determine where to find a given CID, Peer, or IPNS Record on the network;
+  specifically, which network peers provide specific CIDs.
```

Comment:
should delete this line if expanding from content routing to more general routing.